src/Datadog.Trace/Tracer.cs

```diff
@@ -144,6 +145,24 @@ internal static Tracer Create(Uri agentEndpoint, string serviceName, DelegatingH
             return tracer;
         }
 
+        private static Uri DefaultAgentUri()
+        {
+            var prefixes = new string[] { "DATADOG", "DD" };
```

Comment:
If `DD` is the more correct prefix, we should check that one first, in case they have both defined with different values.